Specification/4._CAN_bus_transport_layer.md

```diff
@@ -497,16 +495,15 @@ The following operations are defined for the receiver state:
 Reinitialization includes at least the following:
   * Clearing the transfer payload;
   * Updating the transfer ID value with the actual value from the new transfer;
-  * Clearing the toggle bit;
+  * Clearing the toggle bit; 
   * Initializing the transfer timestamp;
   * Initializing the interface index, if applicable.
 
 The following conditions are defined for the receiver state:
 
-* Uninitialized - this is a default condition, which indicates that the receiver state has not seen any transfer yet,
-and therefore it is considered uninitialized.
-* Transfer ID timeout - last matching transfer has been seen more than 2 seconds ago.
-* Interface switch allowed - this condition is only applicable for configurations with redundant CAN bus interfaces.
+* *Uninitialized* - this is a default condition, which indicates that the receiver has not yet seen any transfers.
+* *Transfer ID timeout* - last matching transfer was seen more than 2 seconds ago.
+* *Interface switch allowed* - this condition is only applicable for configurations with redundant CAN bus interfaces.
 It means that the node is allowed to receive the next transfer from an interface that is not the same as the one the
 previous transfer was received from. The condition will be reached if the last matching transfer has been successfully
```

Comment:
Why use Tswitch as the value here? It renders oddly and is not defined "as a term".

Reply:
It could be replaced by a textual description, but I'm not sure that would make it clearer. You can do that if you think that that would help.

Reply:
@pavel-kirienko Perhaps I wasn't clear - I don't know what this means!

> "matching transfer has been successfully received more than *T<sub>switch</sub>* seconds ago."

What is T switch? Does this have a meaning or could I just replace it with X?

Reply:
You could replace it with X, but personally, I don't think that is necessary.